### PR TITLE
[ZT] browser isolation non-identity

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

**Comment by ranbel** [comment]
Preview: https://ranbel-browser-isolation-non.cloudflare-docs-7ou.pages.dev/cloudflare-one/policies/browser-isolation/agentless/

File: content/cloudflare-one/policies/browser-isolation/agentless/non-identity.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    2. Turn on **Non-identity on-ramp support**.
+4. Create HTTP policies to isolate websites in a remote browser:
+    1. Go to **Gateway** > **Policies** > **HTTP**.
+    2. Build a non-identity [HTTP policy](/cloudflare-one/policies/filtering/http-policies/) using the Isolate action. The following example enables isolation for all web traffic:
+
+        | Selector            | Operator | Value           | Action         |
+        | --------------------| ---------| ----------------| -------------- |
+        | Hostname            | matches regex  | `.*` | Isolate        |
+
+        |Priority|
```
Comment (marciocloudflare) [inline content/cloudflare-one/policies/browser-isolation/agentless/non-identity.md:32]: Is there a reason for Priority to be below the other table? Is it a separate table?

**Review by marciocloudflare — APPROVED** [review]

File: content/cloudflare-one/policies/browser-isolation/agentless/non-identity.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/policies/browser-isolation/agentless/non-identity.md:32]: The settings in the first table are all configured one page, whereas Priority is configured on a separate page. Do you have ideas for a better way to present this info?

Comment (marciocloudflare) [reply content/cloudflare-one/policies/browser-isolation/agentless/non-identity.md:32]: Not really :/

**Pull request merged**